de.dlr.sc.virsat.model.extension.visualisation.ui/src/de/dlr/sc/virsat/model/extension/visualisation/ui/handler/ColorMapPropertyHandler.java

```diff
@@ -63,7 +63,7 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 				
 				if (dialog.open() == Window.OK) {
 					String propertyFQN = dialog.getComparisonProjectPropertyFQN(); 
-					if (vsBaseProject != null && propertyFQN != null) {
+					if (propertyFQN != null) {

```

Comment:
Is the upfront cast in line 58 and 59 already ensuring that there is no null pointer sneaking into the IF scope?

Reply:
Yes, `null instanceof SomeClass` always returns `false`.

https://stackoverflow.com/questions/2950319/is-null-check-needed-before-calling-instanceof